src/fileformats/ocd_file_import.cpp

```diff
@@ -1525,6 +1529,38 @@ void OcdFileImport::setupLineSymbolPointSymbols(OcdFileImport::OcdImportedLineSy
 	}
 }
 
+PointSymbol* OcdFileImport::importSecondarySymbol(const Ocd::LineSymbolCommonV8& attributes, const Ocd::PointSymbolElementV8* elements)
+{
+	if (attributes.secondary_data_size == 0)
+		return nullptr;
+	
+	if (attributes.primary_data_size == 0)
+		return nullptr;  // Secondary symbol handled as primary one in setupLineSymbolPointSymbols
+	
+	auto* symbol = new OcdImportedPointSymbol();
+	const Ocd::OcdPoint32* coords = reinterpret_cast<const Ocd::OcdPoint32*>(elements) + attributes.primary_data_size;
+	setupPointSymbolPattern(symbol, attributes.secondary_data_size, reinterpret_cast<const Ocd::PointSymbolElementV8*>(coords));
+	return symbol;
+}
+
+void OcdFileImport::mergeSecondarySymbol(CombinedSymbol* full_line, LineSymbol* main_line, PointSymbol* secondary_symbol)
+{
+	auto* secondary_line = new OcdImportedLineSymbol();
+	for (auto i = 0u; i < full_line->number_components; ++i)
+		secondary_line->setNumberComponent(i, full_line->getNumberComponent(i));
+	secondary_line->setName(full_line->getName() + tr(" - secondary line"));
+	secondary_line->setMidSymbol(secondary_symbol);
+	secondary_line->setShowAtLeastOneSymbol(false);
```

Comment:
This particular discrepancy is intentional with regard to the MTBO symbol. For short lines , you may get
~~~
|\|
|\/\|
~~~
but not
~~~
|X|
~~~
But I have no idea how it is rendered in the OCAD software, or what particular aspect of #1506 I should look at.

Reply:
I see. The mixed mode makes sense for this particular case.

Ad setting the "at least one mid symbol", see the following comparison:
![image](https://user-images.githubusercontent.com/5584406/119802685-0cffb080-bedf-11eb-9ea8-f3f7f856cb6d.png)

On the left, there is an example rendering of fixed length lines with "at least one mid symbol" set to ON. The inclined brown line indicates the position of a dash point and the number on the left shows the distance of the dash point from the beginning of the line.

As seen in the picture, Mapper squeezes the mid symbol into tiny spaces, which is the aspect of #1506 I intended to raise.

With "at least one mid symbol" set to OFF, the picture is as follows:
![image](https://user-images.githubusercontent.com/5584406/119803790-0faed580-bee0-11eb-8c4e-a736d1ac2266.png)

There are still differences in rendering of the ruined wall and ruined fence, but the other symbols' appearance is similar on both sides. For that reason, I'm in favor of setting "at least one mid symbol" to OFF **on .ocd import** for ISOM maps. The other map standards would have to be evaluated separately.

In the context of this pull request, I advise staying compatible with the current behavior ("at least one" == ON) so that we don't open too many cans of worms at once.